localization/autoware_stop_filter/include/autoware/stop_filter/stop_filter_node.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 TierIV
+// Copyright 2025 TierIV
```

Comment:
Perhaps it looks better to keep `2021` to show from when the code is being licensed :+1: 
```suggestion
// Copyright 2021 TierIV
```